cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringServerHandler.java

```diff
@@ -139,6 +140,53 @@ public void connectControlChannel(FullHttpRequest request, HttpResponder respond
                                    TetheringControlResponse[].class));
   }
 
+  /**
+   * Sends control commands to the client and receives program status updates from the client.
+   */
+  @POST
+  @Path("/tethering/poll/{peer}")
```

Comment:
it's a bit odd to have a verb as part of the url path. Let's use 'channels' instead.

Reply:
fixed